lib/urldata.h

```diff
@@ -1255,6 +1255,8 @@ struct Progress {
   struct timeval t_startop;
   struct timeval t_acceptdata;
 
+  bool is_t_startransfer_set;
```

Comment:
Is it OK to modify the `Progress` struct? And to insert this variable in the middle of the struct? Is there any code (internal or external) that would depend on specific offsets accessing vars here?